marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/jobs.sjs

```diff
@@ -154,7 +155,7 @@ class Jobs {
         timeStarted:  fn.currentDateTime(),
         timeEnded: "N/A",
         hostName: xdmp.hostName(),
-        reqTimeStamp: xdmp.timestampToWallclock(xdmp.requestTimestamp()),
+        reqTimeStamp,
```

Comment:
Should this line be reqTimeStamp: reqTimeStamp ?

Reply:
yeah, for clarity's sake